app/src/main/kotlin/compiler/common/dfa/VariablesOwner.kt

```diff
@@ -0,0 +1,3 @@
+package compiler.common.dfa
```

Comment:
I think this file should be in different directory, as it has nothing to do with DFAs.